contrib/devtools/lint-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check the test suite naming convention
+
+NAMING_INCONSISTENCIES=$(git grep -E '^BOOST_FIXTURE_TEST_SUITE\(' -- \
+    "src/test/**.cpp" "src/wallet/test/**.cpp" | \
```

Comment:
There is also a violation in 
```
src/univalue/test/object.cpp:BOOST_FIXTURE_TEST_SUITE(univalue_tests, BasicTestingSetup)
```
but I realize that is outside the scope of `test_bitcoin` tests. 

Coule make changes upstream to `univalue` as well, but that's minor.

Reply:
Yes, upstreams' tests are intentionally not checked by `lint-tests.sh` since we don't want Travis to block on things where we need help from upstreams. All violations should be possible to fix in-tree :-)